animation_nodes/nodes/gpencil/gp_object_material_output.py

```diff
@@ -0,0 +1,47 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode, VectorizedSocket
+
+class GPObjectMaterialOutputNode(bpy.types.Node, AnimationNode):
```

Comment:
I think this node is not needed. One can use the *Object Material Output* node instead.

Reply:
Okay. Omar, as we have added the option for _Custom Layers_ in _GP Output Object_ node, should we add an option for _Custom Material_ in _Object Material Output_ node? If someone is painting a custom layer, it will also add their custom materials as well.

Reply:
Ok, you may add an Append Material option in the advanced node settings.

Reply:
Okay. I'll make the changes.